code/modules/client/preferences/_preference.dm

```diff
@@ -8,15 +8,18 @@
 /// The priority at which gender is determined, needed for proper randomization.
 #define PREFERENCE_PRIORITY_GENDER 3
 
+/// The priority at which voice is determined, applied after gender for the same reason as PREFERENCE_PRIORITY_BODY_TYPE
```

Comment:
Write out "so we can support the 'use gender' option" explicitly please, your comment is *before* the body type one.

Reply:
Also I don't even see any "use gender" option at all?

Reply:
was something I was planning to implement but forgot about it